provision/roles/node/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: install git and system packages needed by nodejs
+  apt: name={{ item }} state=present
+  become: yes
+  with_items:
+      - libzmq-dev
+      - git-all
+
+- name: download new version of nodejs
+  become: yes
+  get_url:
+    url: http://deb.nodesource.com/setup_6.x
+    dest: /tmp/node_setup_6.x
+    mode: 0755
+
+- name: add node.js repository
+  become: yes
+  shell: /tmp/node_setup_6.x
+
+- name: install new version of nodejs
+  become: yes
+  apt:
+    name: nodejs
+    state: latest
+
+- name: install gulp and bower
+  become: yes
+  shell: npm install -g {{ item }}
```

Comment:
Use an ansible module instead of shell command `npm: name={{ item }} global=yes`